modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/sql/IgniteSqlParserTest.java

```diff
@@ -148,6 +150,51 @@ public void testInvalidDecimalLiteral() {
                 () -> IgniteSqlParser.parse("SELECT decimal '2a'", StatementParseResult.MODE));
     }
 
+    @ParameterizedTest
+    @MethodSource("dmlStatements")
+    public void testDmlStatementsClone(String stmt) {
+        StatementParseResult single = IgniteSqlParser.parse(stmt, StatementParseResult.MODE);
```

Comment:
Seems that similar test for DDL statements fails.
For example  `DROP TABLE TEST` produces the following error:

```
	at org.apache.calcite.util.Util.needToImplement(Util.java:1119)
	at org.apache.calcite.sql.SqlSyntax$7.unparse(SqlSyntax.java:129)
	at org.apache.calcite.sql.SqlOperator.unparse(SqlOperator.java:385)
	at org.apache.calcite.sql.SqlDialect.unparseCall(SqlDialect.java:466)
	at org.apache.calcite.sql.SqlCall.unparse(SqlCall.java:131)
	at org.apache.calcite.sql.SqlNode.toSqlString(SqlNode.java:156)
	at org.apache.calcite.sql.SqlNode.toString(SqlNode.java:131)
```

Is it ok?

Reply:
Thanks, fixed.